AU/Public/Update-Package.ps1

```diff
@@ -120,6 +120,7 @@ function Update-Package {
 
             if ($ChecksumFor -eq 'none') { "Automatic checksum calculation is disabled"; return }
             if ($ChecksumFor -eq 'all')  { $arch = '32','64' } else { $arch = $ChecksumFor }
+            if ($env:au_TrustPackageChecksums -and $global:Latest.ContainsKey('Checksum32') -and $global:Latest.ContainsKey('Checksum64')) { "Automatic checksum validation is disabled"; return }
```

Comment:
Why it needs both checksums to trigger ? It will never work for x64 only tools for example.